### ISSUE TITLE
installtest fails with activate not found (on pypy3 with latest package levels)

### ISSUE THREAD

**Issue by andy-maier** [zhmcclient/python-zhmcclient#673]
`make installtest` fails on pypy3 with PACKAGE_LEVEL=latest.

See Travis runs:
* https://travis-ci.org/github/zhmcclient/python-zhmcclient/jobs/717350936
* https://travis-ci.org/github/zhmcclient/python-zhmcclient/jobs/717350901

```
Testcase test1: Pip install from repo root directory: /home/travis/build/zhmcclient/python-zhmcclient
Saving location of current virtualenv: /home/travis/virtualenv/pypy3.6-7.1.1
Before creating virtualenv: /home/travis/build/zhmcclient/python-zhmcclient/tmp_installtest/virtualenvs/zhmcclient_test_test1
Python version: Python 3.6.1 (784b254d6699, Apr 14 2019, 10:22:42)
[PyPy 7.1.1-beta0 with GCC 6.2.0 20160901] from /home/travis/virtualenv/pypy3.6-7.1.1/bin/python
Pip version: pip 20.2.2 from /home/travis/virtualenv/pypy3.6-7.1.1/site-packages/pip (python 3.6)
Creating virtualenv: /home/travis/build/zhmcclient/python-zhmcclient/tmp_installtest/virtualenvs/zhmcclient_test_test1
Activating virtualenv: /home/travis/build/zhmcclient/python-zhmcclient/tmp_installtest/virtualenvs/zhmcclient_test_test1
Error: Command failed with rc=1: source /home/travis/build/zhmcclient/python-zhmcclient/tmp_installtest/virtualenvs/zhmcclient_test_test1/bin/activate, output follows:
tests/installtest/test_install.sh: line 221: /home/travis/build/zhmcclient/python-zhmcclient/tmp_installtest/virtualenvs/zhmcclient_test_test1/bin/activate: No such file or directory
Makefile:499: recipe for target 'installtest' failed
make: *** [installtest] Error 1
```

**Comment by andy-maier** [comment]
Created issue https://github.com/pypa/virtualenv/issues/1921 for this one.
PR #677 is used to investigate this issue

**Comment by andy-maier** [comment]
Using venv instead of virtualenv works on pypy3. I will change the test_install.sh script to use venv when available (Python 3.6 and higher) and otherwise virtualenv.
